stdsimd/arch/detect/linux/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! Run-time feature detection on Linux
+
+#[cfg(not(any(target_arch = "mips", target_arch = "mips64")))]
+pub mod cpuinfo;
+
+#[cfg(any(target_arch = "aarch64", target_arch = "arm",
+          target_arch = "powerpc64", target_arch = "mips",
+          target_arch = "mips64"))]
```

Comment:
Ideally I'd prefer to avoid the `#[cfg]` here, it becomes a pretty big burden to maintain over time :(